README.md

```diff
@@ -1,371 +1,217 @@
+  </a>
+  <a href="https://travis-ci.org/gatsbyjs/gatsby">
+    <img src="https://travis-ci.org/gatsbyjs/gatsby.svg?branch=master" alt="Current TravisCI build status." />
+  </a>
+  <a href="https://www.npmjs.org/package/gatsby">
+    <img src="https://img.shields.io/npm/v/gatsby.svg?style=flat-square" alt="Current npm package version." />
+  </a>
+  <a href="https://npmcharts.com/compare/gatsby?minimal=true">
+    <img src="https://img.shields.io/npm/dm/gatsby.svg" alt="Downloads per month on npm." />
+  </a>
+  <a href="https://next.gatsbyjs.org/docs/how-to-submit-a-pr/">
+    <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
+  </a>
+</p>
+
+<h3 align="center">
+  <a href="https://next.gatsbyjs.org/docs/">Quickstart</a>
+  <span> · </span>
+  <a href="https://next.gatsbyjs.org/tutorial/">Tutorial</a>
+  <span> · </span>
+  <a href="https://next.gatsbyjs.org/plugins/">Plugins</a>
+  <span> · </span>
+  <a href="https://next.gatsbyjs.org/docs/gatsby-starters/">Starters</a>
+  <span> · </span>
+  <a href="https://next.gatsbyjs.org/docs/how-to-contribute/">Contribute</a>
+  <span> · </span>
+  Support: <a href="https://spectrum.chat/gatsby-js">Spectrum</a>
+  <span> & </span>
+  <a href="https://discord.gg/0ZcbPKXt5bVoxkfV">Discord</a>
+</h3>
+
+## Table of Contents
+
+-   [Get Up and Running in 5 Minutes](#-get-up-and-running-in-5-minutes)
+-   [Learning Gatsby](#-learning-gatsby)
+-   [Migration Guides](#-migration-guides)
+-   [For Advanced Developers: Code Samples & Recipes](#-code-samples-and-recipes)
+-   [How to Contribute](#-how-to-contribute)
+-   [Thanks to Our Contributors and Sponsors](#-thanks-to-our-contributors-and-sponsors)
+
+
+## 🚀 Get Up and Running in 5 Minutes
+
+You can get a new Gatsby site up and running on your local dev environment in 5 minutes with these three steps:
+
+1.  **Create a Gatsby site from a Gatsby starter.**
+
+    Get your Gatsby blog set up in a single command:
+
+    ```sh
+    npx gatsby-cli new my-blazing-fast-site https://github.com/gatsbyjs/gatsby-starter-default#v2
+    ```
+
+2.  **Start the site in `develop` mode.**
+
+    Next, move into your new site’s directory and start it up:
+
+    ```sh
+    cd my-blazing-fast-site/
+    yarn develop
+    ```
+
+    > **NOTE:** If you prefer npm, you can also run `npm run develop` to start the site.
+
+3.  **Open the source code and start editing!**
+
+    Your site is now running at `http://localhost:8000`. Open the the `my-blazing-fast-site` directory in your code editor of choice and edit `src/pages/index.js`. Save your changes and the browser will update in real time!
+
+At this point, you’ve got a fully functional Gatsby website. For additional information on how you can customize your Gatsby site, see our [plugins](https://next.gatsbyjs.org/plugins/) and [the official tutorial](https://next.gatsbyjs.org/tutorial/).
```

Comment:
Let's not link people to next.gatsbyjs.org — it's a temporary thing and we'll be deleting it soon.

Reply:
@jlengstorf and I talked offline and I'm on board to linking to v2 stuff from the README